maxim/models/dataset.py

```
@@ -120,19 +120,19 @@ class DatasetEntry:
-            
+
         Returns:
             DatasetEntry: DatasetEntry object
-            
+
         Raises:
-            TypeError: If data_dict is not a dict   
+            TypeError: If data_dict is not a dict
         """
         if not isinstance(data_dict, dict):
             raise TypeError("data_dict must be a dict")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)